packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.ts

```diff
@@ -77,7 +77,7 @@ const OverrideResolverLoader = function (this: LoaderContext<any>) {
     }
 
     // Lets use the compiler configuration to ensure we are resolving the correct file extensions.
-    const fileExtensions = options?.resolveExtensions
+    const fileExtensions = compiler.options?.resolve?.extensions || options?.resolveExtensions
```

Comment:
This this break if we change the order of these. I think the options could probbaly take precedence. But if there is a default value, we might have to remove it and do something like:

```
const fileExtensions = options?.resolveExtensions || compiler.options?.resolve?.extensions || []
```